.github/workflows/release.yml

```diff
@@ -1,9 +1,34 @@
-name: Publish JS gRPC bindings
+name: Release
 on:
   release:
-    types: [published]
+    types: [created]
 jobs:
-
+  publish_docker:
+    name: Publish Docker Compose Setup
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v1
+      - name: Get latest tag
+        id: latesttag
+        uses: "WyriHaximus/github-action-get-previous-tag@master"
+        env:
+          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+      - name: Update powergate image version
+        run: sed -i.bak 's/latest/${{steps.latesttag.outputs.tag}}/g' docker/powergate-image.yaml
+      - name: Update Makefile
+        run: sed -i.bak 's/powergate-build-context/powergate-image/g' docker/Makefile
```

Comment:
Updates the Makefile running docker-compose to use a tagged powergate image instead of a build context to build from source.

Reply:
Nice, so double magic. First replace `latest`, and then switch context to tagged release. 💯

Reply:
Yep, exactly